src/comps/Navbar/Navbar.jsx

```diff
@@ -1,12 +1,14 @@
 import React  from 'react'
 import './Navbar.css'
 import {useMain} from '../../helpers/context/main-context';
-import {IoIosMenu, IoSearchOutline, CgProfile, BiDotsVerticalRounded, BsMoon, BsSun, IoIosArrowForward, IoSettingsOutline} from '../../utils/getIcons';
+import {IoIosMenu, IoSearchOutline, CgProfile, BiDotsVerticalRounded, BsMoon, BsSun, IoIosArrowForward, IoSettingsOutline, AiOutlineLogout} from '../../utils/getIcons';
 import {StyledButton, StyledDropdown} from '../../styledComps/index';
 import { Link } from "react-router-dom";
+import verifyForm from '../../utils/verifyForm';
 
 function Navbar({toggleDropdownRef}) {
```

Comment:
come on bhai show us some of ur ES6 skills.!!

Reply:
old time preferences